changelog.d/7887.feature

```diff
@@ -0,0 +1 @@
+Add your info here
```

Comment:
I guess we should update the changelog entry with the real entry?

Reply:
[`f8c3001`](https://github.com/vector-im/element-android/pull/7887/commits/f8c3001b6cdc8d74199ce4b6951a43024c80d923)